src/main/scala/nl.knaw.dans.easy.bagstore/BagStoreContext.scala

```diff
@@ -225,7 +225,9 @@ trait BagStoreContext { this: BagFacadeComponent with DebugEnhancedLogging =>
     Files.createDirectory(extractDir)
     val zip = extractDir.resolve("bag.zip")
     FileUtils.copyInputStreamToFile(is, zip.toFile)
-    new ZipFile(zip.toFile).extractAll(extractDir.toAbsolutePath.toString)
+    val zipFile = new ZipFile(zip.toFile)
+    zipFile.setFileNameCharset("UTF-8")
```

Comment:
Same comments apply as in https://github.com/DANS-KNAW/easy-sword2/pull/83#pullrequestreview-33162383